js/script.js

```diff
@@ -1,5 +1,18 @@
 "use strict";
 
+// Mustache
+
+var templateCarousel = document.getElementById("carousel-template").innerHTML;
+var carousel = document.querySelector(".main-carousel");
```

Comment:
staraj sie byc spojnym jesli chodzi cudzyslowy - jesli uzywasz w js pojedynczych, to niech wszedzie beda pojedyncze (oczywiscie za wyjatkiem zagniezdzen czyli np `' " " '`)